content/docs/extensions/model/sklearn.md

```diff
@@ -0,0 +1,69 @@
+# Scikit-Learn Models Support
+
+[ModelType](/doc/user-guide/mlem-abcs#modeltype) implementations for any
+sklearn-compatible classes as well as `Pipeline`
```

Comment:
Product questions (rhetorical i.e. these need links or better explanations IMO):
- What are sklearn-compatible classes ?
- What is `Pipeline`?

However, this intro is a bit low-level (even for a reference). It's OK to be specific about what the extension is/does but can we try to zoom out, use more general language, etc? Something like "Enables you to connect Scikit-Learn classes to MLEM by implementing a model object..." (just an idea)